RecoTauTag/Configuration/python/boostedHPSPFTaus_cfi.py

```diff
@@ -19,8 +19,11 @@ def addBoostedTaus(process):
     process.combinatoricRecoTausBoosted.modifiers.remove(process.combinatoricRecoTausBoosted.modifiers[3])
     #process.combinatoricRecoTausBoosted.builders[0].pfCandSrc = cms.InputTag('pfNoPileUpForBoostedTaus')
     process.combinatoricRecoTausBoosted.builders[0].pfCandSrc = cms.InputTag('particleFlow')
-    process.hpsPFTauDiscriminationByLooseMuonRejection3Boosted.dRmuonMatchLimitedToJetArea = cms.bool(True)
-    process.hpsPFTauDiscriminationByTightMuonRejection3Boosted.dRmuonMatchLimitedToJetArea = cms.bool(True)
+    #Note JetArea is not defined for subjects and restiction to jetArea is turned to dRMatch=0.1, so better use the latter explicitely
```

Comment:
BTW, same applied to RecoTauTag/RecoTau/python/PFRecoTauDiscriminationAgainstMuon2_cfi.py l.25